### DISCUSSION TITLE
Does Prisma follow semver?

### DISCUSSION THREAD

**Discussion by zardoy** [prisma/prisma#5308] [Q&A] [answered]
I'm just wondering why Prisma doesn't bump the major version of `@prisma/client` every time when Prisma introduces a Breaking Change? For example user could use `Prisma.FindUniqueUserArgs` for convenience and with minor update he got a broken code :( Sorry if I missed something

**Comment by zardoy** [comment, marked as answer]
@williamluke4 Could you help with that?

**Reply by williamluke4** [reply]
Hey @zardoy, this should explain everything much better than I can.
- [How does Prisma versioning work](https://www.prisma.io/docs/about/releases#how-does-prisma-versioning-work)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
